lib/api.js

```diff
@@ -122,8 +123,8 @@ PlexAPI.prototype._request = function _request(relativeUrl, method, parseRespons
             );
         }
 
-        if (response.statusCode !== 200) {
-            return deferred.reject(new Error('Plex Server didnt respond with status code 200, response code: ' + response.statusCode));
+        if (response.statusCode < 200 && response.statusCode > 299) {
```

Comment:
Found what's probably causing the test to fail, `statusCode` cannot be both below 200 **and** over 299.

Replacing `&&` with `||` seems to fix it :smile: